ios/fastlane/Appfile

```diff
@@ -1,5 +1,5 @@
 app_identifier("net.givtapp.kids") # The bundle identifier of your app
-apple_id("ppavelko.job@gmail.com") # Your Apple Developer Portal username
+apple_id("p.pavelko.job@gmail.com") # Your Apple Developer Portal username
```

Comment:
This file was added in the previous xcode config PR, I wonder if we actually need it for non local ci/cd

Reply:
akaik the email is needed for provisioning profiles regeneration via match (with pushing them to the https://github.com/givtnl/givt.apps.codesigning repo)